tests/harness/executor/executor.go

```diff
@@ -34,12 +35,12 @@ func main() {
 	}
 }
 
-func run(parallelism int, goFlag bool, gogoFlag bool, ccFlag bool, javaFlag bool, pythonFlag bool) (successes, failures, skips uint64) {
+func run(parallelism int, goFlag bool, gogoFlag bool, ccFlag bool, javaFlag bool, pythonFlag bool, externalFlag string) (successes, failures, skips uint64) {
```

Comment:
This seems like a good opportunity to generalize `run()` over a list of harness instead of adding another flag. Then the external harness can be treated just like the rest and get appended to the list of targets.

Reply:
> Thanks, this looks good. I'm hoping we can generalize some of the function calls so we don't need to pass so many arguments around.

Thanks for the quick review. I added a commit that pushes up the building of the Harness list up to main, and this indeed eliminates wiring this flag down the call stack.